Windows Tests actions fail

<!-- Thanks for submitting an issue! Please describe the issue. -->

# Problem

Windows Tests actions fail during the "Install dependencies" step - i.e. [this one](https://github.com/giorgiobasile/prefect-planetary-computer/actions/runs/5832758908).

# Traceback / Example

```
Getting requirements to build editable: started
  Getting requirements to build editable: finished with status 'error'
  error: subprocess-exited-with-error
  
  Getting requirements to build editable did not run successfully.
  exit code: 1
  
  [[21](https://github.com/giorgiobasile/prefect-planetary-computer/actions/runs/5832758908/job/15818800384#step:4:22) lines of output]
  Traceback (most recent call last):
    File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 353, in <module>
      main()
    File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 335, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 132, in get_requires_for_build_editable
      return hook(config_settings)
    File "C:\Users\runneradmin\AppData\Local\Temp\pip-build-env-d5o5hepf\overlay\Lib\site-packages\setuptools\build_meta.py", line 450, in get_requires_for_build_editable
      return self.get_requires_for_build_wheel(config_settings)
    File "C:\Users\runneradmin\AppData\Local\Temp\pip-build-env-d5o5hepf\overlay\Lib\site-packages\setuptools\build_meta.py", line 341, in get_requires_for_build_wheel
      return self._get_build_requires(config_settings, requirements=['wheel'])
    File "C:\Users\runneradmin\AppData\Local\Temp\pip-build-env-d5o5hepf\overlay\Lib\site-packages\setuptools\build_meta.py", line 3[23](https://github.com/giorgiobasile/prefect-planetary-computer/actions/runs/5832758908/job/15818800384#step:4:24), in _get_build_requires
      self.run_setup()
    File "C:\Users\runneradmin\AppData\Local\Temp\pip-build-env-d5o5hepf\overlay\Lib\site-packages\setuptools\build_meta.py", line 487, in run_setup
      super(_BuildMetaLegacyBackend,
    File "C:\Users\runneradmin\AppData\Local\Temp\pip-build-env-d5o5hepf\overlay\Lib\site-packages\setuptools\build_meta.py", line 338, in run_setup
      exec(code, locals())
    File "<string>", line 12, in <module>
    File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\encodings\cp1[25](https://github.com/giorgiobasile/prefect-planetary-computer/actions/runs/5832758908/job/15818800384#step:4:26)2.py", line 23, in decode
      return codecs.charmap_decode(input,self.errors,decoding_table)[0]
  UnicodeDecodeError: 'charmap' codec can't decode byte 0x8d in position 2[39](https://github.com/giorgiobasile/prefect-planetary-computer/actions/runs/5832758908/job/15818800384#step:4:40)4: character maps to <undefined>
  [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

Getting requirements to build editable did not run successfully.
exit code: 1

See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
Error: Process completed with exit code 1.
```

- [ ] I would like to [help contribute](https://giorgiobasile.github.io/prefect-planetary-computer/#contributing) a pull request to resolve this!

--- a/setup.py
+++ b/setup.py
@@ -8,7 +8,7 @@ with open("requirements.txt") as install_requires_file:
 with open("requirements-dev.txt") as dev_requires_file:
     dev_requires = dev_requires_file.read().strip().split("\n")
 
-with open("README.md") as readme_file:
+with open("README.md", encoding="utf-8") as readme_file:
     readme = readme_file.read()
 
 setup(

